ddtrace/tracer/sqlcomment.go

```diff
@@ -57,6 +57,8 @@ const (
 	sqlCommentDBService     = "dddbs"
 	sqlCommentParentVersion = "ddpv"
 	sqlCommentEnv           = "dde"
+	sqlCommentPeerHostname  = "ddh"
+	sqlCommentPeerDBName    = "dddb"
```

Comment:
should this be `dddbn` to match the naming convention of `sqlCommentDBService` above?

Reply:
This is a cross-tracer compatibility question; `dddb` is used in other tracers at this point in time =/

Reply:
😢 Okay - no worries.